documentation/src/object-initializer.md

```diff
@@ -0,0 +1,32 @@
+# Initializer
+
+## Index Initializers
+
+Use index initializers with dictionary.
+This elegantly prevents uninitialized dictionaries.
+
+[Object and Collection Initializers (C# Programming Guide)]: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#object-initializers-with-collection-read-only-property-initialization
+
+```csharp
+var dict = new Dictionary<string, int>
+{
+    ["key1"] = 1,
+    ["key2"] = 50,
+};
+```
+
+*Not so good example*
```

Comment:
The captions should be *after* the corresponding code block so that they're stylised correctly.
See: https://github.com/messerli-informatik-ag/code-style/blob/310e3d99d14fae55629928e5945adca3fd929b28/documentation/custom.css#L6-L12

Reply:
Weird, but I should have learned it from existing files.
c6018200d4232e68cafe17c654ec66de6b7d2a68